uber/templates/regform.html

```diff
@@ -136,7 +136,7 @@
             $.each($("input:radio[name='amount_extra']"), function() {
                 $(this).parents('.btn').removeClass('active btn-primary');
                 $(this).parents('.btn').addClass('btn-default');
-                if ($(this).val() <= $.val('amount_extra')) {
+                if ($(this).val() == $.val('amount_extra')) {
                     $(this).parents('.btn').removeClass('btn-default');
```

Comment:
I can't review this super closely right now but I saw most of this earlier,
+1

On Wed, Aug 10, 2016, 11:04 Dominic Cerquetti <notifications@github.com>
wrote:

> In uber/templates/regform.html
> <https://github.com/magfest/ubersystem/pull/1946#discussion_r74285937>:
>
> > @@ -136,7 +136,7 @@
> >              $.each($("input:radio[name='amount_extra']"), function() {
> >                  $(this).parents('.btn').removeClass('active btn-primary');
> >                  $(this).parents('.btn').addClass('btn-default');
> > -                if ($(this).val() <= $.val('amount_extra')) {
> > +                if ($(this).val() == $.val('amount_extra')) {
> >                      $(this).parents('.btn').removeClass('btn-default');
>
> has the effect of not selecting all previous tiers when the selection
> changes, just this one
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/magfest/ubersystem/pull/1946/files/32a5450e1ec45261cd7c259300a9bd24a46728ba#r74285937>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AG3WB7B2ddfS5Ovg72EjA6fLQFXizD2jks5qegSagaJpZM4JgwDl>
> .
>